src-browserify/core/model.js

```diff
@@ -1,23 +1,15 @@
-(function() {
-
-  cdb._debugCallbacks= function(o) {
-    var callbacks = o._callbacks;
-    for(var i in callbacks) {
-      var node = callbacks[i];
-      console.log(" * ", i);
-      var end = node.tail;
-      while ((node = node.next) !== end) {
-        console.log("    - ", node.context, (node.context && node.context.el) || 'none');
-      }
-    }
-  }
```

Comment:
couldn't find any usage of `cdb._debugCallbacks` so :fire: 
let me know if I should put it back